sdk/tests/wallet/claim_outputs.rs

```diff
@@ -128,15 +128,15 @@ async fn claim_2_basic_outputs_no_outputs_in_claim_account() -> Result<()> {
 
     let token_supply = account_0.client().get_token_supply().await?;
     let rent_structure = account_0.client().get_rent_structure().await?;
-    let expiration_time = account_0.client().get_time_checked().await? + 86400; // 1 Day from now
+    let expiration_slot = account_0.client().get_slot_index().await? + 86400;
```

Comment:
Add TODOs just because this is weird to leave like this?

Reply:
Done